roles/confluent.common/defaults/main.yml

```
@@ -54,14 +54,9 @@ jolokia_jar_url: "http://search.maven.org/remotecontent?filepath=org/jolokia/jol
 ### Full URL used for Prometheus Exporter Jar Download
 jmxexporter_jar_url: https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.12.0/jmx_prometheus_javaagent-0.12.0.jar
 
-### Boolean to have cp-ansible download the Confluent CLI
-confluent_cli_download_enabled: "{{rbac_enabled}}"
-
-### Full path on hosts to install the Confluent CLI
-confluent_cli_path: /usr/local/bin/confluent
-
 ### A path reference to a local archive file or URL. By default this is the URL from Confluent's repositories. In an ansible-pull deployment this could be set to a local file such as "~/.ansible/pull/{{inventory_hostname}}/{{confluent_archive_file_name}}".
-confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}-{{confluent_archive_scala_version}}.tar.gz"
-
+# confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}-{{confluent_archive_scala_version}}.tar.gz"
+## Uncomment above line and change back for 6.0 release
+confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}.tar.gz"
```

```suggestion
confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}.tar.gz"
```

remove commented code
also run `python doc.py` to update VARIABLES.md and commit that